examples/resources/spectrocloud_application_profile/resource.tf

```diff
@@ -1,37 +1,67 @@
-resource "spectrocloud_application_profile" "test_profile_tf"{
-  name = "profile-herbs"
-  version = "1.0.0"
-  context = "project"
-  tags = ["sivaa", "terraform"]
+resource "spectrocloud_application_profile" "app_profile_all_tiers" {
+  name        = "profile-all-tiers-test"
+  version     = "1.0.0"
+  context     = "project"
+  tags        = ["sivaa", "terraform"]
   description = "test"
-  cloud = "all"
+  cloud       = "all"
+  # Sample Container Tier
   pack {
-    type = "operator-instance"
-    name = "mysql-3-stage"
-    source_app_tier="636c0714c196e565df7a7b37"
+    name            = "container-tier"
+    type            = "container"
+    registry_uid    = "63245b43e52f17068237e0cd"
+    source_app_tier = "636223d44ead10f3bdff62d9"
+    values          = "pack:\n  namespace: \"{{.spectro.system.appdeployment.tiername}}-ns\"\n  releaseNameOverride: \"{{.spectro.system.appdeployment.tiername}}\"\npostReadinessHooks:\n  outputParameters:\n    - name: CONTAINER_SVC\n      type: lookupSecret\n      spec:\n        namespace: \"{{.spectro.system.appdeployment.tiername}}-ns\"\n        secretName: \"{{.spectro.system.appdeployment.tiername}}-custom-secret\"\n        ownerReference:\n          apiVersion: v1\n          kind: Service\n          name: \"{{.spectro.system.appdeployment.tiername}}-svc\"\n        keyToCheck: metadata.name\n    - name: CONTAINER_SVC_PORT\n      type: lookupSecret\n      spec:\n        namespace: \"{{.spectro.system.appdeployment.tiername}}-ns\"\n        secretName: \"{{.spectro.system.appdeployment.tiername}}-custom-secret\"\n        ownerReference:\n          apiVersion: v1\n          kind: Service\n          name: \"{{.spectro.system.appdeployment.tiername}}-svc\"\n        keyToCheck: spec.ports[0].port\n        keyFormat: string, number\ncontainerService:\n  serviceName: \"{{.spectro.system.appdeployment.tiername}}-svc\"\n  registryUrl: \"\"\n  image: alphine\n  access: public\n  ports:\n    - \"8080\"\n  serviceType: LoadBalancer\n  args:\n    - $TEST\n  command:\n    - sh\n    - ./start.sh\n  env:\n    - name: TEST\n      value: \"true\"\n  volumeName: TestVolume\n  volumeSize: 10\n  pathToMount: /pack\n"
+  }
+  # Sample Helm Tier
+  pack {
+    name            = "kafka-tier"
+    type            = "helm"
+    registry_uid    = "605f66e748cd5ef5bd070ba8"
+    source_app_tier = "638f21ac7f63fb9677625873"
```

Comment:
We should reference a data source in the example instead of hardcoding values. This will help customers do the right thing.

Reply:
update with datasource variable